README.md

```diff
@@ -19,6 +19,40 @@ Python module for reading and writing UNTL metadata records.
   <title qualifier="officialtitle">This is the title for the record</title>
 </metadata>
 ```
+
+
+Requirements
+-------------
+
+* Python ~= 2.7
+
+
+Installation
+--------------
+
+This application can be installed by following the steps below:
+
+    $ git clone https://github.com/unt-libraries/pyuntl.git
+    
+    $ cd pyuntl
+
+Install the dependencies for this project from root of project using:
```

Comment:
I think it might be more correct to delete this line, as the next command is still part of the installation process. `python setup.py install` doesn't _just_ install the dependencies, it also installs the applications itself.